Misc/NEWS.d/next/Library/2025-04-29-11-48-46.gh-issue-132876.lyTQGZ.rst

```diff
@@ -0,0 +1 @@
+Workaround broken ``ldexp()`` implementation on Windows 10.
```

Comment:
```suggestion
Workaround broken ``ldexp()`` implementation on Windows 10, which fail to
round subnormal results.
```

a variant of news entry.

Reply:
"fails", but otherwise looks good to me.

Reply:
Suggest instead (tone, grammar, and spelling things out for plain end users):

>  `ldexp()`` on Windows doesn't round subnormal results before Windows 11., but should. Python's wrapper now does round them, so results may change slightly, in rare cases of very small results, on Windows versions before 11.

Note that It's not specific to Windows 10, and people still run Python on earlier Windows despite that we don't officially support them. "The truth, the whole truth, and nothing but the truth" :wink: